commit 057754ed3c0bd9fc4a4661e9484554fda6a02960 (refs/changes/64/1564/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-23T15:57:24-08:00 (2 years, 7 months ago)
    
    DOC-691 update Manta/remora-themed docs logo at top-right

diff --git a/deps/restdown-brand-remora b/deps/restdown-brand-remora
index f0fd8ff..26e1938 160000
--- a/deps/restdown-brand-remora
+++ b/deps/restdown-brand-remora
@@ -1 +1 @@
-Subproject commit f0fd8ff870fa28c851e209c16cd2f90305990ddf
+Subproject commit 26e1938181be79e71ed7469f165184aa05c4f5b2
